components/ChallengeMaterial/__snapshots__/ChallengeMaterial.test.js.snap

```diff
@@ -623,16 +623,18 @@ exports[`Curriculum challenge page Should render first challenge that is not pas
               class="border border-lightgray rounded my-1"
             >
               <div
-                class="ms-3"
+                class="mx-3"
```

Comment:
`mx` will add a margin horizontally (left & right). This will make it look consistent because the right side will have a margin too.